core/io/translation_loader_po.cpp

```diff
@@ -110,6 +116,9 @@ RES TranslationLoaderPO::load_translation(FileAccess *f, Error *r_error, const S
 		}
 
 		if (l == "" || l.begins_with("#")) {
+			if (l.ends_with("fuzzy")) {
```

Comment:
I've never worked with PO files before. I should have read the spec first, didn't know it had one. I thought it was Weblate specific. :P